ios-swift/App/Realm.plist

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>baseUrl</key>
+	<string>https://realm.mongodb.com</string>
```

Comment:
I don't think we use this outside of the GraphQL app, but it should also include the project/app id in the path since you can't access the Realm UI with only the App ID™️.

e.g. `https://realm.mongodb.com/groups/57879f6ce4b04cba440ba8c5/apps/5b366d043cd49d6e3add621e/`

Reply:
For baseUrl on iOS it's just the domain I believe.